nimble/host/src/ble_hs_pvcy.c

```diff
@@ -64,6 +64,12 @@ ble_hs_pvcy_remove_entry(uint8_t addr_type, const uint8_t *addr)
 {
     struct ble_hci_le_rmv_resolve_list_cp cmd;
 
+    /* First check if privacy started flag is enabled, no point in going
+     * further if pvcy_started is not enabled */
+    if (!ble_hs_pvcy_started) {
```

Comment:
ble_hs_prvcy_started  means only that rpa timeout has been set in the controller and Controller will keep generating local RPA.
If timeout is disabled we can still use resolving list.
It is usually a case when NImble use public address but still wants to use resolving list.

Therefore I believe that this change will actually break Nimble when it not uses RPA.

Reply:
Thank you for the clarification !!